File: WebAPI/Data/Models/Person.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;
using WebAPI.Utilities.Attributes;

namespace WebAPI.Data.Models
{
    public class Person : Model
    {
        public enum GenderType {
            Male,
            Female,
            Non
        }

        public string FullName { get; set; } = null!;
        public DateTime DOB  { get; set; }
        public string PhoneNumber { get; set; } = null!;
        public string? Email { get; set; }
        public GenderType Gender { get; set; }
    }
}

File: WebUI/react-ui/src/models/person.ts
import { Dayjs } from "dayjs";
import { Model } from "./model";

export enum GenderType {
    male,
    female,
    non
}

export default class Person extends Model {
    constructor(
        public id: number,
        public fullName: string,
        public DOB: Dayjs,
        public phoneNumber: string,
        public gender: GenderType,
        public email?: string
    ) {
        super(id);
    }
}